references/obj_detection/train_pytorch.py

```diff
@@ -79,9 +79,6 @@ def evaluate(model, val_loader, metric):
 
 
 def main(args):
-
-    print(args)
-
```

Comment:
Actually that one is on purpose so that we always have a console output displaying the training setup :sweat_smile:

Reply:
Oh I see Thanks:)